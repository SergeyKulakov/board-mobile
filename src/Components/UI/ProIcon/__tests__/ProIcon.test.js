import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { ProIcon as Component } from '../index'

describe('ProIcon', () => {
  it('render snapshot', () => {
    const wrapper = shallow(<Component isVisible />)
    const wrapper2 = shallow(<Component isVisible={false} />)

    expect([wrapper, wrapper2]).toMatchSnapshot()
  })
})
