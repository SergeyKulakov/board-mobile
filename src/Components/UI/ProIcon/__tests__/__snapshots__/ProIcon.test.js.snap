// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProIcon render snapshot 1`] = `
Array [
  <Styled(View)>
    <Styled(Icon) />
  </Styled(View)>,
  "",
]
`;
